test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -708,6 +708,38 @@ def test_successful_subsequent_recurring_cof_transaction
     assert_successful_response(response)
   end
 
+  def test_missing_Field
+    @options = @options.merge({
+      address: {
+        address1: 'Unspecified',
+        city: 'Unspecified',
+        state: 'NC',
+        zip: '00000',
+        country: ''
+      }
+    })
+
+    assert response = @gateway.purchase(@amount, @credit_card, @options)
+    assert !response.success?
```

Comment:
There's a bit of a cleaner way to do this without the negation:
```suggestion
    assert_failure response
```

Reply:
https://github.com/activemerchant/active_merchant/blob/63233b7e6c85f785d1636a6da0a1b42eda1ea2bd/test/remote/gateways/remote_cyber_source_test.rb#L723

https://github.com/activemerchant/active_merchant/blob/63233b7e6c85f785d1636a6da0a1b42eda1ea2bd/test/remote/gateways/remote_cyber_source_test.rb#L739